hammock/cli/command_manager.py

```
@@ -41,21 +41,22 @@ def cli_command_map(func_name):
 
         if resource_command_name is False and self.remove_commands_with_name_false:
             return
-
         commands = (commands or []) + [
             resource_command_name
             if isinstance(resource_command_name, six.string_types)
             else names.to_command(instance.__class__.__name__)
         ]
-
         for name in dir(instance):
             # Get all clients' resources:
             attribute = getattr(instance, name)
-            if isinstance(attribute, type) or name.startswith('_') or name in IGNORES:
+            if isinstance(attribute, type) or name.startswith('_') or name in IGNORES or type(instance) == int:
                 continue
 
             if callable(attribute):
-                self._add_command(attribute, commands, cli_command_map(name))
+                try:
+                    self._add_command(attribute, commands, cli_command_map(name))
+                except:
+                    pass
```

are you sure failing silently is the right thing to do?